catroid/src/org/catrobat/catroid/ui/dialogs/OverwriteRenameMediaDialog.java

```diff
@@ -218,8 +218,6 @@ private boolean handleOkButton() {
 					for (LookData lookData : ProjectManager.getInstance().getCurrentSprite().getLookDataList()) {
 						if (lookData.getLookName().compareTo(mediaName) == 0) {
 							lookToRemove = lookData;
-							ProjectManager.getInstance().getCurrentSprite().getLookDataList().remove(lookData);
```

Comment:
Please have another look at this. I think this can be implemented more elegantly. For example: you could just remove the look here rather than in lines 223 - 226, and add a break in order to prevent the loop from iterating through all looks after the correct one has been found (same goes for the sounds: lines 235 - 238).